servant/src/Servant/API.hs

```diff
@@ -19,6 +19,8 @@ module Servant.API (
   -- | Retrieving the HTTP version of the request
   module Servant.API.QueryParam,
   -- | Retrieving parameters from the query string of the 'URI': @'QueryParam'@
+  module Servant.API.Fragment,
+  -- | Retrieving parameter from the fragment of the 'URI': @'Fragment'@
```

Comment:
Hmm, now you can't /retrieve/ anything, maybe instead write here "Documenting the fragment part of the URI: ..." ?

Reply:
Fixed.